index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <title> Weather Report </title>
+    <link rel="stylesheet" href="styles/index.css">
+<body>
+    <div class="center-wrap bg">
+        <header class="header header-box">
+            <h1>Weather Report</h1>
+            <span>Welcome to the city of 
+                <span id="header-city">
+                    San Francisco
+                </span>
+            </span>
+        </header>
+        <br>
+        <div class="weather-box center-wrap">
+            <h2>Weather Garden</h2>
+            <div id="weatherContent" class="weather-content">
+                <div id="sky">☁️     ☁️   ☁️ ☀️ ☁️  ☁️</div>
+                <br>
+                <br>
+                <br>
```

Comment:
instead of adding multiple breaks take advantage of CSS here. Using padding or margin to create the space you want.